### DISCUSSION TITLE
Frontend Edit of Category Descriptions

### DISCUSSION THREAD

**Discussion by aarogge** [joomla/joomla-cms#28118] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
I have editors who update the wording on a website.  They only work from the frontend. They are unable to edit the category descriptions.

### Describe the solution you'd like
I would like the ability to edit the text of category descriptions from the frontend. I don't want any other permissions from the frontend.

### Additional context
I am not a writer.  I depend upon others for content. When my editors can't update text, it falls to me. I would like to be concerned with the structure and design and let others deal with the text.

I am running 3.9.5

**Comment by jwaisner** [comment]
Feature requests are not available for J3, however this can be looked at for J4. Tagging request for J4.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28118">issues.joomla.org/tracker/joomla-cms/28118</a>.</sub>

**Comment by AndySDH** [comment]
Good idea.
